scripts/types.js

```diff
@@ -7,11 +7,25 @@ export const generateTypes = async (symbols, dir) => {
   const file = path.resolve(dir, 'index.d.ts');
   console.log(`Generating ${path.relative('', file)}`);
   const versions = await getVersions(symbols);
-  const types = getTypes(Object.keys(versions));
+  const types = getTypes(Object.keys(versions), symbols);
   await fs.writeFile(file, types);
   console.log('Done');
 };
 
-const getTypes = (icons) => {
-  return 'TODO';
+const getTypes = (icons, isSymbol) => {
+  const lines = [];
+  const formattedIcons = icons.map(icon => `  "${icon}"`);
+  const typeNames = isSymbol ? ['MaterialSymbols', 'MaterialSymbol'] : ['MaterialIcons', 'MaterialIcon'];
```

Comment:
Nit: Please use a stricter check `isSymbol === true`. Since it is read from command line, I'm checking if it is a boolean true wherever it is used.